src/dashboard/Data/CustomDashboard/elements/ViewElement.react.js

```
@@ -0,0 +1,330 @@
+      <div className={styles.noData}>
+        <Icon name="visibility" width={32} height={32} fill="#64748b" />
+        <p>No data found</p>
+      </div>
+    );
+  }
+
+  // Get columns from data if not specified
+  const displayColumns = columns || Object.keys(data[0]).filter(k => k !== 'ACL');
+
+  // Calculate column widths based on content
+  const columnWidths = useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)